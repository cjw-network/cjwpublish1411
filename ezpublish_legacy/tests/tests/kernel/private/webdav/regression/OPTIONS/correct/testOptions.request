<?php

// Test if OPTIONS returns correct information

$_SERVER['REQUEST_URI'] = '*';
$_SERVER['REQUEST_METHOD'] = 'OPTIONS';

?>
