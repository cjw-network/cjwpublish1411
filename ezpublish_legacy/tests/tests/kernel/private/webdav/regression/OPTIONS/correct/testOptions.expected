DAV: 1
Allow: GET, HEAD, PROPFIND, PROPPATCH, OPTIONS, DELETE, COPY, MOVE, MKCOL, PUT, LOCK, UNLOCK
Server: eZComponents/dev/ezcWebdavTransportWrapper
Content-Length: 0
HTTP/1.1 200 OK
